dev/README_RELEASE_PROVIDER_PACKAGES.md

```diff
@@ -493,6 +510,13 @@ cd "${AIRFLOW_REPO_ROOT}"
 breeze build-docs apache-airflow-providers cncf.kubernetes sftp --clean-build
 ```
 
+Alternatively, if you set the environment variable: `PACKAGE_LIST` above, just run the command:
```

Comment:
@amoghrajesh - proposal (at least that's how most past approaches have been handled so far). The PACKAGE_LIST is not discoverable other than via  reading the docs and code. How about defining it as another option in the commands ?

```
@click.option(
    "--package-list",
    help="Optional, contains comma-separate list of packag ids that are selected for package preparation and document publishing. You. can use it instead of individually adding the packages as arguments.",
    type=str,
    envvar='PACKAGE_LIST'
)
```

This. way you will see it as option when you run `--help` (you will need to add it to `_config.py`  as well in appropriate section) and you will not have specifically read them with os.getenv (because `envvar` is doing it for you).

Reply:
This is much more consistent with all the other parameters we pass to the commands.